packaging/rpm/ice.spec

```diff
@@ -0,0 +1,854 @@
+#
+# Copyright (c) ZeroC, Inc. All rights reserved.
+#
```

Comment:
This is mostly a copy of the 3.7 ice.spec from the ice-packaging repository.

Removed 3.8 unsupported platforms and features but still requires more work.